public_html/nextflow_tutorial.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$title = 'Nextflow tutorial';
+$subtitle = 'Learn how nf-core pipelines are automatically kept up to date with community standards.';
```

Comment:
Not sure if the subtitle describes the tutorial. I would probably go for something along the lines of:
"Learn the basics of Nextflow and how to write your own Nextflow pipelines."

Reply:
Yeah, copy paste error sorry